src/routes/adm/classes/getAllClasses.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @memberof module:router~mainRouter~admRouter
+ * @inner
+ * @namespace class/getAllClasses
+ */
+const { Classes } = require('../../../models/classes')
+
+/**
+ * Main register function
+ * @name GET /adm/classes
+ * @function
+ * @memberof module:router~mainRouter~admRouter~classes/getAllClasses
+ * @inner
+ * @async
+ * @param {Object} req
+ * @param {Object} res
+ * @returns 200 if OK
+ * @returns 500 if Internal Server Error
+ */
+module.exports = async (req, res) => {
+  try {
+    const classes = await Classes.find()
```

Comment:
This gets all classes, from all facilities. Limit it to the classes from users's facility